train.gemspec

```diff
@@ -19,11 +19,11 @@ Gem::Specification.new do |spec|
   spec.test_files    = spec.files.grep(%r{^(test|spec|features)/})
   spec.require_paths = ['lib']
 
-  spec.required_ruby_version = '>= 2.0'
+  spec.required_ruby_version = '>= 2.3'
 
   spec.add_dependency 'json', '>= 1.8', '< 3.0'
   # chef-client < 12.4.1 require mixlib-shellout-2.0.1
```

Comment:
Do we need this comment anymore?

Reply:
dunno, i don't really understand anything about that comment or what it was important enough of an observation to merit (which meant i didn't want to touch it)